src/Sylius/Bundle/MoneyBundle/composer.json

```diff
@@ -29,7 +29,7 @@
         "sylius/resource-bundle": "^1.11",
         "symfony/framework-bundle": "^6.4.1 || ^7.0",
         "symfony/intl": "^6.4.0 || ^7.0",
-        "symfony/templating": "^6.4.0",
+        "symfony/templating": "^6.4.0 || ^7.0",
```

Comment:
we should remove that dependency. there's no 7.0 release.

Reply:
I'll make that on another PR.

Reply:
I've removed it here: https://github.com/Sylius/Sylius/pull/16918